pkg/generic_advdiff/GAD_OPTIONS.h

```diff
@@ -19,18 +19,16 @@ C Use this file for selecting options within the GAD package
 #ifdef ALLOW_GENERIC_ADVDIFF
 C     Package-specific Options & Macros go here
 
-C This flag selects the form of COSINE(lat) scaling of bi-harmonic term.
-C *only for use on a lat-lon grid*
-C Setting this flag here only affects the bi-harmonic tracer terms; to
-C use COSINEMETH_III in the momentum equations set it CPP_OPTIONS.h
+C This flag selects the form of COSINE(lat) scaling of horizontal
+C bi-harmonic diffusivity. *only for use on a lat-lon grid*
+C Setting this flag here only affects tracer diffusivity; to use it
+C in the momentum equations it needs to be set in MOM_COMMON_OPTIONS.h
```

Comment:
I find this confusion, because first it is bi-harmonic diffusivity and in the next line it is just diffusivity. I would also leas the "only for use ..." in a separate line.
```suggestion
C *only for use on a lat-lon grid*
C This flag selects the form of COSINE(lat) scaling of horizontal
C bi-harmonic diffusivity. 
C Setting this flag here only affects tracer (bi-harmonic) diffusivity; to use it
C in the momentum equations it needs to be set in MOM_COMMON_OPTIONS.h
```

Reply:
I have the impression that the risk of confusion is minimal here: tracer biharmonic diffusion is part of tracer diffusion, so it's not wrong. Here the main point is to stress that momentum is not affected (although the CPP option has the same name); and we don't repeat/mention neither other restriction (lat-lon grid,  cosPower >0, ...).